packit_service/worker/handlers/bodhi.py

```diff
@@ -133,12 +135,22 @@ def run(self) -> TaskResults:
             issue_repo = self.service_config.get_project(
                 url=self.job_config.issue_repository
             )
-            body = (
-                f"Bodhi update failed to be created for `{self.koji_build_event.nvr}`:\n"
-                "```\n"
-                f"{ex}\n"
-                "```"
-            )
+
+            if isinstance(ex.__cause__, AuthError):
+                body = (
+                    f"Bodhi update failed to be created for `{self.koji_build_event.nvr}` "
+                    f"because of the missing permissions.\n\n"
+                    f"Please, give {self.service_config.fas_user} user `commit` rights in the "
+                    f"[dist-git settings]({self.data.project_url}/adduser)."
+                )
+            else:
+                body = (
+                    f"Bodhi update failed to be created for `{self.koji_build_event.nvr}`:\n"
```

Comment:
dtto

Reply:
Do you consider an issue that we will create a new issue for new occurrences (e.g. for pitty..) since the title will not match anymore?